vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItemFactory.kt

```diff
@@ -239,6 +192,7 @@ class RoomSummaryItemFactory @Inject constructor(
             .hasDraft(roomSummary.userDrafts.isNotEmpty())
             .itemLongClickListener { _ -> onLongClick?.invoke(roomSummary) ?: false }
             .itemClickListener { onClick?.invoke(roomSummary) }
+            .roomType(RoomUtils.getRoomType(roomSummary))
 
     private fun createCenteredRoomSummaryItem(
```

Comment:
it seems this is a new item, we should verify it is correctly displayed in Tchap and if we need to pass the room type as for the other item

Reply:
It's a tchap item.  @Florian14 you can see it at line 195.
@Claire1817 please restore the tchap comment